extensions/azurecore/package.json

```diff
@@ -259,6 +263,11 @@
           "command": "azure.resource.startterminal",
           "when": "viewItem == azure.resource.itemType.account",
           "group": "azurecore"
+        },
+        {
+          "command": "azure.resource.openInAzurePortal",
+          "when": "isDatabaseOrServer",
```

Comment:
@Charles-Gagnon @abist I'm still really bad when it comes to this. What's a good setting here and for the group?

We want this to only show up if azureTenantId and azureResourceId is defined on a connection profile

Reply:
There are 2 cases here. The azureTenantId and azureResourceId is set - 
1. compile time - then you can just use a boolean which is defined in the class
2. dynamically in run time - use `vscode.commands.executeCommand('setContext', 'azureTenantId', true);`

Reply:
Not sure what you mean by "what a good setting here" is. But from what I think you're saying you would likely be looking to modify https://github.com/microsoft/azuredatastudio/blob/master/src/sql/workbench/services/objectExplorer/browser/mssqlNodeContext.ts#L31

Reply:
Yeah thinking on this, there isn't really a way for this option to show up without this information being populated anyway.